backend/src/models/income.ts

```diff
@@ -0,0 +1,10 @@
+export type Income = {
+  income_id: number;
+  user_id: number;
+  amount: number;
+  income_date: Date;
+  summary: string | null;
+};
+
+export type CreateIncomePayload = Omit<Income, 'user_id' | 'income_id'>;
```

Comment:
Used for when sending a post request to create a income, the payload has some attributes of income not all. Omit just takes the type and removes the fields that you specify